Add a new source to the English names

octopusinvitro opened the pull request:
The new source was added because Ibrahim Najjar was disappearing from `morph/wikidata.csv`, due to him being removed from the source for English names. Tony found a new Wikidata source where he is present so I added it as a second source for English names.

tmtmtmtm commented on scraper.rb:
```diff
@@ -6,11 +6,17 @@
+en_names2 = EveryPolitician::Wikidata.wikipedia_xpath(
+  url: "https://en.wikipedia.org/wiki/Category:Members_of_the_Parliament_of_Lebanon",
+  xpath: '//div[contains(@class, "mw-category-group")]//li/a[not(@class="new")]/@title'
+)
```
Ah, you don't need to scrape the page directly like this. You just need:

```
en_names2 = WikiData::Category.new( 'Category:Members of the Parliament of Lebanon', 'en').member_titles
```

See docs at https://github.com/everypolitician/wikidata-fetcher

octopusinvitro committed:
Add a new source to the English names

The new source was added because Ibrahim Najjar was disappearing from morph/wikipdata.csv, due to him being removed from the source for English names. Tony found a new Wikidata source where he is present, so I added it as a second source for English names.

octopusinvitro commented:
@tmtmtmtm Since I was force-pushing anyway to remove the weird line, I added the feedback from your comment.

I checked that the warnings we get are [the same we were getting at morph.io](https://morph.io/tmtmtmtm/lebanese-parliament-wikidata#console_title), but we are getting a new one for [**Abdel Latif El Zein**](https://en.wikipedia.org/wiki/Abdel_Latif_El_Zein):

```bash
Can't find Wikidata IDs for: Abdel Latif El Zein in en
‼ Q1995 is_instance_of year. Skipping
No data for Q1995
‼ Q2591 is_instance_of determinator for date of periodic occurrence. Skipping
No data for Q2591
‼ Q2535 is_instance_of determinator for date of periodic occurrence. Skipping
No data for Q2535
‼ Q1990 is_instance_of leap year & year. Skipping
No data for Q1990
‼ Q5109530 is_instance_of political party. Skipping
No data for Q5109530
```

tmtmtmtm commented:
👍 

Not sure what the problem with Abdel Latif El Zein is, but that seems like a separate issue to be investigated another time…
